ietf-document/draft-quic-4-sat/draft-kuhn-quic-4-sat-04.txt

```diff
@@ -0,0 +1,728 @@
+
+   o  Emulated packet loss on both downlink and uplink paths:
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 7]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+      *  Uniform random transmission losses: 1%, or
+
+      *  Bursty losses, including transmission and congestion losses:
+         P(g|g)= 0.982, P(g|b)= 0.645, P(b|b)= 0.355, P(b|g)= 0.018 (see
+         [RFC6534] for more details)
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
+   During the transmission of 100 MB on the download path, the test
+   should report the downloading time of 2 MB, 10 MB and 100 MB.
+
+5.  Mechanisms that improve the performance of QUIC for SATCOM
+
+5.1.  Getting up to speed
+
+   The advantage of using QUIC is that it includes the TLS and TCP
+   negotiations that reduce the time at which the data can be
+   transmitted.  That being said, results of [IJSCN19] illustrate that
+   it will take many RTTs for the congestion controller to increase the
+   rate before it fills the bottleneck capacity.  This dominates
+   performance when a path has a large RTT (as in GEO SATCOM networks).
+   There is an issue in QUIC getting up to speed in a SATCOM context.
+
+   The tuning of the initial window described in
+   [I-D.irtf-iccrg-sallantin-initial-spreading] which has been shown to
+   improve performance both for high BDP and more common BDP
+   [CONEXT15][ICC16] could be a relevant solution.  That being said,
+   such solution requires the usage of pacing to avoid important bursts
+   of packets in a network that does not have a large BDP.
+
+5.2.  Maximum window
+
+   A large number of in-flight packets are prewired to fully exploit the
+   bottleneck capacity, when there is a large BDP.  Default values of
+   maximum windows may not be suitable for a SATCOM context.
+
+   Such as presented in [PANRG105], only increasing the initial
+   congestion window is not the only way that can improve QUIC
+   performance in a SATCOM context: increasing maximum congestion
+   windows can also result in much better performance.  Other protocol
+   mechanisms also need to be considered, such as flow control at the
+   stream level in QUIC.
+
+
+
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 8]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+5.3.  Reliability
+
+   Packet loss detection and loss repair take additional time on paths
+   with a larger RTT.  This increases the time that a server needs to
+   react to a congestion event.  Both can impact the user experience.
+   This happens when a user uses a Wi-Fi link to access a SATCOM
+   terminal.  Although the benefits need to weighed against the
+   additional capacity in introducing end-to-end FEC and the potential
+   to use link-local ARQ and/or link-adaptive FEC.  End-to-end
+   connections may not only suffer from losses in the Wi-Fi segment but
```

Comment:
The pair of words /may not/ is prone to misinterpretation. 
I suggest:
/End-to-end connections may  suffer from losses not only in the Wi-Fi segment, but.../